crates/uv/tests/it/lock_conflict.rs

```diff
@@ -4413,11 +4413,17 @@ conflicts = [
         "#,
     )?;
 
-    // I believe there are multiple valid solutions here, but the main
-    // thing is that `x2` should _not_ activate the `idna==3.4` dependency
-    // in `proxy1`. The `--extra=x2` should be a no-op, since there is no
-    // `x2` extra in the top level `pyproject.toml`.
+    // Error out, as x2 extra is only on the child.
```

Comment:
Not sure if the test is still relevant as this now errors out, but since there's also a lock assertion right after, I've added a `sync` call without argument below, as I wasn't sure.